dozer-types/src/models/api_endpoint.rs

```diff
@@ -82,8 +223,8 @@ pub struct ApiEndpoint {
     pub path: String,
     #[prost(message, tag = "4")]
     pub index: Option<ApiIndex>,
+
     #[prost(message, tag = "5")]
```

Comment:
Do you think we should keep this `skip_serializing_if`?

Reply:
Yeah, added this field again